CMakeLists.txt

```diff
@@ -254,21 +254,17 @@ else()
     set(PROGNAME keepassxc)
 endif()
 
-if(APPLE AND WITH_APP_BUNDLE AND "${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr/local")
-    set(CMAKE_INSTALL_PREFIX "/Applications")
-    set(CMAKE_INSTALL_MANDIR "/usr/local/share/man")
-endif()
-
 if(MINGW)
     set(CLI_INSTALL_DIR ".")
     set(PROXY_INSTALL_DIR ".")
     set(BIN_INSTALL_DIR ".")
     set(PLUGIN_INSTALL_DIR ".")
     set(DATA_INSTALL_DIR "share")
 elseif(APPLE AND WITH_APP_BUNDLE)
-    set(CLI_INSTALL_DIR "/usr/local/bin")
```

Comment:
While this solves the permissions issue, it would be interesting to have a menu option to link the cli binaries to /usr/local/bin like some mac apps usually do:

![image](https://user-images.githubusercontent.com/107672/50501739-9aeebd00-0a41-11e9-875f-d2ce13c8adc9.png)

Reply:
Yes that is a great solution and exposes the command line tools as an option. This can be made available in 2.5